### PR TITLE
Reword Credential nickname prompt and placeholder

### LINKED ISSUE #2265: Reword prompt for Credential Nickname
https://positpbc.slack.com/archives/C05D7NZD52S/p1726065183528269

Image

The Title Case is awkward here, and the “Credential” concept was not previously introduced. Consider saying something a little more casual like:
Enter a unique nickname for this server and API key combination.

### PR DESCRIPTION
**Pull request by dotNomad**
A nice and simple reword for the Credential `multiStepInputs`:

From: Enter a Unique Nickname for your Credential.
To: Enter a unique nickname for this server and API key combination.

`Example: ` was also removed from the placeholder text to make it more similar to other VS Code inputs.

## Intent

Resolves #2265

### CODE REVIEW

File: extensions/vscode/src/multiStepInputs/newCredential.ts [hunk 1]
```diff
@@ -246,8 +246,9 @@ export async function newCredential(
       step: thisStepNumber,
       totalSteps: state.totalSteps,
       value: currentName,
-      prompt: "Enter a Unique Nickname for your Credential.",
-      placeholder: "example: Posit Connect",
+      prompt:
+        "Enter a unique nickname for this server and API key combination.",
+      placeholder: "Posit Connect",
```
Comment (jonkeane) [inline extensions/vscode/src/multiStepInputs/newCredential.ts:251]: Technically this is _just_ server — right? We still only accept/allow one key per server?

Comment (sagerb) [reply extensions/vscode/src/multiStepInputs/newCredential.ts:251]: you are correct @jonkeane . Perhaps we should change this to `Enter a unique nickname for this server`

File: extensions/vscode/src/multiStepInputs/newDeployment.ts [hunk 2]
```diff
@@ -691,8 +691,9 @@ export async function newDeployment(
         step: 0,
         totalSteps: 0,
         value: currentName,
-        prompt: "Enter a Unique Nickname for your Credential.",
-        placeholder: "example: Posit Connect",
+        prompt:
+          "Enter a unique nickname for this server and API key combination.",
+        placeholder: "Posit Connect",
```
Comment (sagerb) [inline extensions/vscode/src/multiStepInputs/newDeployment.ts:696]: My comment above applies here as well: `Enter a unique nickname for this server`

**Review by sagerb — APPROVED** [review]
Minor changes requested, but feel free to merge after you've made them.

**Pull request merged**